retinaface/commons/preprocess.py

```diff
@@ -33,6 +33,31 @@ def resize_image(img, scales, allow_upscaling):
 
     return img, im_scale
 
+def preprocess_batch_images(batch_images):
+    images = []
+    img_scales = []
+    scales = [1980, 1024]
+    for image in batch_images:
+        img_h, img_w = image.shape[0:2]
+        scale_x = scales[0] / img_w
+        scale_y = scales[1] / img_h
+        scale = scale_x
+        if scale_y<scale_x<1 or 1<scale_y<scale_x or scale_y<1<scale_x:
```

Comment:
Thanks for sharing this set of changes! One note I've run into testing it out here.

This could default to `scale_x` in exact match scenarios.
```python
if scale_y<scale_x<=1 or 1<=scale_y<scale_x or scale_y<=1<=scale_x:
```
should be used instead.

Reply:
Yes, you are right.